src/common.h

```diff
@@ -29,11 +29,6 @@ typedef uint64_t u64;
 #include "target.h"
 #include "std.h"
 
-//FATFS is defined by target_defs.h
-struct FAT {
-    char inaccessible[FATSTRUCT_SIZE];
-};
-
```

Comment:
It was made opaque to ensure that no code goes snooping inside the FAT structure, since this is supposed to be a 'private' structure.  Maybe less relevant now that we have devofs which has a different structure.